witchcraft-server/src/logging/mod.rs

```diff
@@ -59,10 +64,43 @@ pub(crate) async fn init(
     let request_logger = logger::appender(install, metrics, hooks).await?;
     let audit_logger = logger::appender(install, metrics, hooks).await?;
 
+    // ignore if the audit logger is already filled
```

Comment:
I'd panic if the logger is already set personally.

Reply:
That's fair! Why panic vs returning `Err`?

Reply:
It could return an error, but this method is purely internal and is only called during startup - if it's called twice then it's a programmer error like indexing out of bounds.